ramalama/model_store/store.py

```diff
@@ -126,6 +126,44 @@ def get_snapshot_hash(self, model_tag: str) -> str:
     def get_snapshot_directory_from_tag(self, model_tag: str) -> str:
         return os.path.join(self.snapshots_directory, self.get_snapshot_hash(model_tag))
 
+    def resolve_model_argument(
```

Comment:
I think cli/input details such as `dry_run` shouldn't be leaked into the store.

Reply:
Can remove this method, it's not called.